requirements.txt

```diff
@@ -8,6 +8,7 @@ botocore==1.6.1
 cachetools==2.0.1
 certifi==2017.7.27.1
 cffi==1.10.0
+chained-aws-lambda==0.0.3
```

Comment:
https://pypi.python.org/pypi/chained-aws-lambda publishes 0.0.3 but there are no tags in https://github.com/chanzuckerberg/chained-aws-lambda, making it hard to review the release. Consider using https://github.com/HumanCellAtlas/data-store-cli/blob/master/common.mk to cut releases with automatic tagging and annotation.

(Also, not urgent or blocking, but the docs in that repo need more updating/cleanup)

Reply:
I bumped to v0.0.5 which has a tag.

I agree, the docs are in terrible shape. :(